cypress/e2e/emailValidation.cy.js

```diff
@@ -0,0 +1,22 @@
+import registrationPage from '/Users/tetianazholudova/Documents/CypressTest/cypress/support/pages/RegistrationPage.js'
+import registrationPageStepsEmailField from '../support/steps/registrationPageStepsEmailField.js'
+
+describe('Email validation', () => {
+    beforeEach(() => {
+    
+      cy.visit('/')
+      cy.get('.hero-descriptor_btn').click()
+  
+    })
+
+it('should show "Email required" when the email field is empty', ()=> {
+    registrationPage.getSignupEmail().click();
+    registrationPageStepsEmailField.leaveEmailFieldEmpty()
```

Comment:
it is overhead

Reply:
for this scenario two cypress methods would be enough 

https://docs.cypress.io/api/commands/blur
https://docs.cypress.io/api/commands/focus